kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue

```diff
@@ -8,6 +8,25 @@
     </template>
     <template v-else>
       <b-container class="mb-2 mt-4 no-container-padding">
+        <template v-if="hasNodesForSection('featured-channel')">
+          <b-container>
+            <h5 class="mt-2 text-muted">
+              {{ $tr('channelLabel') }}
+            </h5>
+          </b-container>
+          <SlidableGrid
+            v-slot="slotProps"
+            :nodes="sectionNodes['featured-channel']"
+          >
+            <ChannelCard
```

Comment:
I think this is now called `EkChannelCard`.

Reply:
Ouch! I failed to rebase this, sorry. Fixed now.